garage.gemspec

```diff
@@ -16,7 +16,7 @@ Gem::Specification.new do |s|
   s.files = Dir["{app,config,db,lib}/**/*"] + ["MIT-LICENSE", "Rakefile", "README.md"]
 
   s.add_dependency "rails", '>= 4.0.0'
-  s.add_dependency "doorkeeper", ">= 0.6.7", "< 1.5.0"
+  s.add_dependency "doorkeeper", "~> 2.1.4"
```

Comment:
It seems [doorkeeper gem has `doorkeeper_authorize!` from v2.0.0](https://github.com/doorkeeper-gem/doorkeeper/blob/master/NEWS.md#200---2014-12-16). Are there any reason to define dependency as `~> 2.1.4`? I think defining  `>= 2.0.0` is enoght for the incompatible `doorkeeper_for` change.